src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -96,27 +92,5 @@ private JObject InvokeScript(JArray _params)
             CheckWitnessHashes checkWitnessHashes = _params.Count >= 2 ? new CheckWitnessHashes(((JArray)_params[1]).Select(u => UInt160.Parse(u.AsString())).ToArray()) : null;
             return GetInvokeResult(script, checkWitnessHashes);
         }
-
-        [RpcMethod]
-        private JObject GetUnclaimedGas(JArray _params)
```

Comment:
This method was removed?

Reply:
@joeqian10  Move to https://github.com/neo-project/neo-modules/pull/273?